scripts/build/postpublish.js

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env node
+
+// Resets the package json to the original state, which was modified during
+// prepublish phase
+
+const fs = require('fs');
+
+const { cwd } = process;
+
+const pkgPath = `${cwd()}/package.json`;
+const pkgPathTmp = `${cwd()}/.tmp.package.json`;
+
+// No need to safeguard here cause if there is an error, the process gets aborted
+// and node closes all open and temporary files, ergo node takes care of the cleanup
+fs.copyFileSync(pkgPathTmp, pkgPath);
+
+fs.unlinkSync(pkgPathTmp);
```

Comment:
I'm not a fan of those synced methods, but I would at least here suggest to use the normal method. It makes no sense to "block until done" here, because the workflow is finished after that anyways. :)

Reply:
I think there is no diff and anyway is good to have process blocking in case of errors